extensions/standard-processors/processors/HashContent.h

```diff
@@ -42,7 +43,8 @@
 
 using HashReturnType = std::pair<std::string, int64_t>;
 
-namespace {
+// Without puttng this into its own namespace, the code would export already defined symbols.
+namespace { // NOLINT
```

Comment:
Note: we should eventually fix this by marking the implementations `inline` or by moving them to impl files, and not using static linkage.

Reply:
Agreed and noted here:
https://issues.apache.org/jira/browse/MINIFICPP-1258